emr-user-role-mapper-application/src/test/com/amazon/aws/emr/integration/utils/IAMUtils.java

```diff
@@ -0,0 +1,87 @@
+package com.amazon.aws.emr.integration.utils;
+
+import com.amazonaws.services.identitymanagement.AmazonIdentityManagement;
+import com.amazonaws.services.identitymanagement.AmazonIdentityManagementClientBuilder;
+import com.amazonaws.services.identitymanagement.model.AttachRolePolicyRequest;
+import com.amazonaws.services.identitymanagement.model.CreatePolicyRequest;
+import com.amazonaws.services.identitymanagement.model.CreatePolicyResult;
+import com.amazonaws.services.identitymanagement.model.CreateRoleRequest;
+import com.amazonaws.services.identitymanagement.model.CreateRoleResult;
+import com.amazonaws.services.identitymanagement.model.DeletePolicyRequest;
+import com.amazonaws.services.identitymanagement.model.DeleteRoleRequest;
+import com.amazonaws.services.identitymanagement.model.DetachRolePolicyRequest;
+import com.amazonaws.services.identitymanagement.model.EntityAlreadyExistsException;
+import com.amazonaws.services.identitymanagement.model.GetPolicyRequest;
+import com.amazonaws.services.identitymanagement.model.GetRoleRequest;
+import com.amazonaws.services.identitymanagement.model.Policy;
+import com.amazonaws.services.identitymanagement.model.Role;
+
+/**
+ * Utility methods for AWS IAM.
+ */
+public class IAMUtils {
+
+  final static AmazonIdentityManagement iam =
+      AmazonIdentityManagementClientBuilder.defaultClient();
+
+
+  public static Policy createPolicy(String account, String policy) {
```

Comment:
it looks like for any logged user, only one integ test run can live at a given time. If multiple users logged in as the same user, then test runs may interfere with each other given the policies and mapping file would be exactly the same. This may not be a blocker for now. But if it is true, we may want to document this limitation.

Reply:
The role and policies have user name added to them. The policy name has the hashcode appened which is a functioon of username - so I think multiple users can run the tests.

Reply:
Yea.. multiple users can run tests in parallel but not same user I guess. It's ok for now.